jobs/ccg-plugin/spec

```diff
@@ -0,0 +1,51 @@
+---
+name: ccg-plugin
+
+templates:
+  pre-start.ps1.erb: bin/pre-start.ps1
+  post-stop.ps1.erb: bin/post-stop.ps1
+  credential_spec.json.erb: config/credential_spec.json
+
+packages:
+- ccg-plugin
+
+properties:
+  ccg.enabled:
+    description: When enabled, winc will pass a credential spec to HCS on container creation.
+    default: false
+
+  ccg.plugin_guid:
+    description: The curly-brace-enclosed GUID of the COM class which implements the ICcgDomainAuthCredentials::GetPasswordCredentials method. See https://docs.microsoft.com/en-us/windows/win32/api/ccgplugins/nf-ccgplugins-iccgdomainauthcredentials-getpasswordcredentials.
+    default: "{8019A64C-3F4E-4DE3-AD2B-9A544290E2C3}"
+
+  ccg.plugin_input:
```

Comment:
@jrussett It'd be nice to have an example of this plugin input. If I remember correctly, it needed to be `:` separated input with a domain/username/password or something similar